ui/component/copyableStreamkey/view.jsx

```diff
@@ -0,0 +1,66 @@
+// @flow
+import * as ICONS from 'constants/icons';
+import { FormField } from 'component/common/form';
+import Button from 'component/button';
+import React, { useRef } from 'react';
+
+type Props = {
+  copyable: string,
+  snackMessage: ?string,
+  doToast: ({ message: string }) => void,
+  label?: string,
+  primaryButton?: boolean,
+  name?: string,
+  onCopy?: (string) => string,
+};
+
+export default function CopyableText(props: Props) {
```

Comment:
Rename to actual function name for clarity

Reply:
Renamed the function